handlers/products.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strconv"
 	"text/template"
 
-	"github.com/joaog/API_Test/models"
+	"github.com/JoaoGumiero/Crud/models"
```

Comment:
Did you see it here? We are importing the package using the name that we defined in the beginning, using the `go mod init` stuff

Reply:
Wow, that's good for me to know. Act, at the begining i was struggling on how to import packages, but now I get it.